src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParsing.scala

```diff
@@ -113,34 +116,43 @@ trait SlickExpressionParser extends JavaTokenParsers {
   private def entityRootDot: Parser[RootFunc] =
     "this." ^^ { _ => entityRootFunc}
 
+  private val namespacedIdent: Parser[String] = rep1sep(ident, ":") ^^ {
```

Comment:
The `rep1sep` parser will match strings like `one:two:three` but it will then not match the cases below. I haven't tested the implications of this yet.

Could also possibly write this as a `Parser[AttributeName]` ? Again, not sure of the implications.